assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/blip/mlflow_wrapper.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""MLflow PythonModel wrapper class that loads the MLflow model, preprocess inputs and performs inference."""
+
+import mlflow
+from PIL import Image
+import pandas as pd
+import torch
+import tempfile
+
+from transformers import AutoProcessor, Blip2ForConditionalGeneration
+from config import MLflowSchemaLiterals, MLflowLiterals, Tasks
+from typing import List, Tuple
+
+try:
+    # Use try/except since vision_utils is added as part of model export and not available when initializing
+    # model wrapper for save_model().
+    from vision_utils import create_temp_file, process_image, get_current_device
+except ImportError:
+    pass
+
+
+class BLIPMLFlowModelWrapper(mlflow.pyfunc.PythonModel):
```

Comment:
From what I understand this is to support Blip2 model only. Since, there exists another model named Blip (without suffix 2, i.e. the 1st version). Should we rename Blip to Blip2 every where (including name of parent folder of this file) ?